### DISCUSSION TITLE
Source code of Laragon.exe binary ?

### DISCUSSION THREAD

**Discussion by theokeist** [leokhoa/laragon#510] [Q&A] [answered]
Hi,, is there a repo for standalone Laragon.exe file ? All that is possible to change now is through the folders & config files..
I would like to deep down into Laragon itself, the look & feel 😸

**Comment by Pen-y-Fan** [comment, marked as answer]
The source code for laragon.exe isn't available. Leo has previously given his reasons including security. It's unlikely Leo will change his mind.

This question is another reason:  What language is it developed in? #89

**Reply by theokeist** [reply]
Yeah that make sense to me now, thx for answer.. 
Pascal/Delphi would be nice to look at :-)
